croutonjs/src/js/crouton-core.js

```diff
@@ -395,7 +395,7 @@ function Crouton(config) {
 
 	self.getServiceBodies = function (service_bodies_id, callback) {
 		jQuery.getJSON(this.config['root_server'] + '/client_interface/jsonp/?switcher=GetServiceBodies'
-			+ getServiceBodiesQueryString(service_bodies_id) + '&callback=?', callback);
+			+ getServiceBodiesQueryString(service_bodies_id) + '&parents=1&callback=?', callback);
```

Comment:
I'm wondering if there's any reasonable way to make the addition of `parents=1` here conditional, because it'll increase the size of the response payload for everyone. Could be significant for folks who are using tomato.

Reply:
good point, yeah it should only do that if using one of the values that requires this data.  I'll make the changes here.